spec/loggability/logger_spec.rb

```diff
@@ -156,10 +156,57 @@
 			expect( logger.logdev.instance_variable_get(:@shift_size) ).to eq( 125000 )
 		end
 
+		it "can be told to log to a file and delegate to ruby's built-in logger log device" do
+			logfile = double( "logfile.log" )
+			expect( Loggability::LogDevice ).to receive( :create ).
+				with( :file, 'log_file.log' ).
+				and_return( logfile )
+
+			logfile = Loggability::LogDevice.create( :file, 'log_file.log' )
+			expect( logger ).to receive( :output_to ).
+				with( logfile ).
+				and_return( nil )
+
+			logger.output_to( logfile )
+
+			expect( logger.logdev ).to be_a( Logger::LogDevice )
+		end
+
+		it "can be told to log to datadog and not send messages" do
```

Comment:
This spec description to me reads like you can configure datadog to not send messages, but what's being tested is that it doesn't post until hitting the batch size. Would you mind adding something about the batch size to these two descriptions?